src/Hazelcast.Net/DistributedObjects/IHMap.Removing.cs

```diff
@@ -25,12 +26,15 @@ public partial interface IHMap<TKey, TValue> // Removing
         /// within the specified time to wait value.
         /// </summary>
         /// <param name="key">A key.</param>
-        /// <param name="timeToWait">The time to wait for a lock on the key.</param>
+        /// <param name="timeToWait">How long to wait for a lock on the key (<see cref="TimeToWait.Infinite"/> i.e. -1ms to wait forever;
```

Comment:
We need to update the documentation. It is mentioning deleted files.

Reply:
ok